app/controllers/review_apps_controller.rb

```diff
@@ -52,11 +52,11 @@ def permitted_params
                     :image_name,
                     :image_tag,
                     :before_deploy,
-                    environment: [
-                      :name,
-                      :value,
-                      :ssm_path,
-                      :value_from
+                    environment: %i[
+                      name
+                      value
+                      ssm_path
```

Comment:
Seems confusing to mix leading-`:` symbols with `%i[]`-style symbols. I almost thought there we were calling a method called `name`.

Reply:
good point!
just disabled it in https://github.com/degica/barcelona/pull/678/commits/b28362c463e81c20aab258e84455ce13b056a236